mbu/module1/src/main/java/com/epam/jmp/fileshare/dao/impl/DiskFileShareDaoImpl.java

```diff
@@ -0,0 +1,180 @@
+package com.epam.jmp.fileshare.dao.impl;
+
+import com.epam.jmp.fileshare.dao.FileShareDao;
+import com.epam.jmp.fileshare.dto.FileDto;
+import com.epam.jmp.fileshare.exceptions.FileShareException;
+import com.epam.jmp.fileshare.util.FileShareDateFormatter;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import java.io.*;
+import java.text.MessageFormat;
+import java.util.*;
+
+/**
+ * Created by nbuny on 20.07.2016.
+ */
+@Repository
+public class DiskFileShareDaoImpl implements FileShareDao {
+
+    private static final String NAME_PROPERTY_POSTFIX = ".name";
+    private static final String EXTENSION_PROPERTY_POSTFIX = ".extension";
+    private static final String EXPIRATION_PROPERTY_POSTFIX = ".expiration";
+
+    private static final String PROPERTY_FILE_EXTENSION = ".properties";
+    private static final String REPOSITORY_LOCATION = "repositoryLocation";
+    private static final String REPOSITORY_NAME = "repositoryName";
+    private static final String DEFAULT_FILE_EXPIRATION_PERIOD = "defaultFileExpirationPeriod";
+    @Autowired
+    @Qualifier("disk-dao.properties")
+    private Properties defaultRepositoryProperties;
+    private Properties repositoryProperties;
+
+    private String repositoryLocation;
+    private String repositoryName;
+    private int defaultFileExpirationPeriod;
```

Comment:
The value of the field DiskFileShareDaoImpl.defaultFileExpirationPeriod is not used